src/tasky/__main__.py

```diff
@@ -494,26 +470,29 @@ def ReadLoop(tasky):
         tasky.PrintAllTaskLists()
 
     # Convert all input to unicode type with utf-8 encoding.
-    readIn = unicode(raw_input(USAGE), 'utf-8')
+    #readIn = unicode(raw_input(USAGE), 'utf-8')
     # shlex does not accept unicode types, so convert to str before lexing.
     # Also prepend a string to hold the place of the application name.
-    args = [''] + shlex.split(readIn.encode('utf-8'))
+    args = [''] + shlex.split(input())
     # Decode back to unicode type again before further processing.
-    args = [x.decode('utf-8') for x in args]
+    #args = [x.decode('utf-8') for x in args]
 
     # Re-populate flags based on this input.
-    FLAGS.Reset()
+    #FLAGS.Reset()
     FLAGS(args)
 
     if FLAGS.quit:
       break
     tasky.HandleInputArgs()
 
 
-def main(args):
+def main(args=None):
+  if args is None:
+    args = sys.argv
+
   # Ensure that stdout is written as utf-8.
-  writer = codecs.getwriter('utf-8')
-  sys.stdout = writer(sys.stdout)
+  # writer = codecs.getwriter('utf-8')
```

Comment:
Is this not needed anymore? If it still is, I guess there's a py3 idiom for it: https://stackoverflow.com/Questions/4374455/how-to-set-sys-stdout-encoding-in-python-3